cmd/apps.go

```diff
@@ -50,7 +50,7 @@ func (d DeisCmd) AppCreate(id, buildpack, remote string, noRemote bool) error {
 
 	if !noRemote {
 		if err = git.CreateRemote(s.Client.ControllerURL.Host, remote, app.ID); err != nil {
-			if err.Error() == "exit status 128" {
+			if strings.Contains(err.Error(), fmt.Sprintf("fatal: remote %s already exists.", remote)) {
```

Comment:
Unfortunately all git errors fall under a generic `gitError` type so we cannot tell the difference between a `git remote already exists` error and a `not a git repository` error.